integration_tests/tests/sqs.rs

```diff
@@ -22,86 +25,119 @@ fn list_queues() {
 
 #[test]
 fn sqs_roundtrip_tests() {
-	let _ = env_logger::try_init();
```

Comment:
Converted from tabs to spaces here. Also needed to move from time-based queue names to random-based ones due to race conditions where both tests would run within the same second.